tests/cli/test_revpath.py

```diff
@@ -111,12 +105,11 @@ def test_can_parse(self):
             self.assertEqual(None, project)
 
         with self.subTest("dataset (no context)"):
-            with self.assertRaises(WrongRevpathError) as cm:
-                parse_full_revpath(dataset_url)
-            self.assertEqual(
-                {ProjectNotFoundError, MultipleFormatsMatchError},
-                set(type(e) for e in cm.exception.problems)
-            )
+            dataset, project = parse_full_revpath(dataset_url)
+            if project:
+                scope_add(project)
+            self.assertTrue(isinstance(dataset, IDataset))
+            self.assertEqual(None, project)
```

Comment:
I changed it, because there are no errors anymore. Since the Datumaro format can now be uniquely identified, the dataset is successfully loaded despite no format being specified.

Reply:
Try to add a test for the previously existing situation.

Reply:
Okay, I added a new test using a more ambiguous dataset.